docs/guide/xlang_type_mapping.md

```diff
@@ -4,4 +4,90 @@ sidebar_position: 3
+| timestamp          | 18      | Instant         | datetime             | Number          | std::chrono::nanoseconds       | Time             | DateTime         |
+| decimal            | 19      | BigDecimal      | Decimal              | bigint          | /                              | /                | /                |
+| binary             | 20      | byte[]          | bytes                | /               | `uint8_t[n]/vector<T>`         | `[n]uint8/[]T`   | `Vec<uint8_t>`   |
+| array              | 21      | array           | np.ndarray           | /               | /                              | array/slice      | Vec              |
+| bool_array         | 22      | bool[]          | ndarray(np.bool_)    | /               | `bool[n]`                      | `[n]bool/[]T`    | `Vec<bool>`      |
+| int8_array         | 23      | byte[]          | ndarray(int8)        | /               | `int8_t[n]/vector<T>`          | `[n]int8/[]T`    | `Vec<i18>`       |
+| int16_array        | 24      | short[]         | ndarray(int16)       | /               | `int16_t[n]/vector<T>`         | `[n]int16/[]T`   | `Vec<i16>`       |
+| int32_array        | 25      | int[]           | ndarray(int32)       | /               | `int32_t[n]/vector<T>`         | `[n]int32/[]T`   | `Vec<i32>`       |
+| int64_array        | 26      | long[]          | ndarray(int64)       | /               | `int64_t[n]/vector<T>`         | `[n]int64/[]T`   | `Vec<i64>`       |
+| float16_array      | 27      | float[]         | ndarray(float16)     | /               | `fury::float16_t[n]/vector<T>` | `[n]float16/[]T` | `Vec<fury::f16>` |
+| float32_array      | 28      | float[]         | ndarray(float32)     | /               | `float[n]/vector<T>`           | `[n]float32/[]T` | `Vec<f32>`       |
+| float64_array      | 29      | double[]        | ndarray(float64)     | /               | `double[n]/vector<T>`          | `[n]float64/[]T` | `Vec<f64>`       |
+| tensor             | 30      | /               | /                    | /               | /                              | /                | /                |
+| sparse tensor      | 31      | /               | /                    | /               | /                              | /                | /                |
+| arrow record batch | 32      | /               | /                    | /               | /                              | /                | /                |
+| arrow table        | 33      | /               | /                    | /               | /                              | /                | /                |
+
+# Type info(not implemented currently)
+
+Due to differences between type systems of languages, those types can't be mapped one-to-one between languages.
+
+If a user sees one type has multiple fury types, for example, `long` in java has type `int64/varint64/sliint64`, it
```

Comment:
I suggest changing it to the following:

If the user notices that one language type corresponds to multiple Fury types

Reply:
thanks, updated